package-lock.json

```diff
@@ -8,6 +8,7 @@
       "name": "spacify",
       "version": "0.1.0",
       "dependencies": {
+        "axios": "^1.3.4",
```

Comment:
Never commit package-lock.json, it gets build from package.json. Also could you delete this file altogether?

Reply:
Yup. Unstaged it.